apps/emqx_management/src/emqx_mgmt_api_cluster.erl

```diff
@@ -188,19 +285,43 @@ running_cores() ->
     Running = emqx:running_nodes(),
     lists:filter(fun(C) -> lists:member(C, Running) end, emqx:cluster_nodes(cores)).
 
-invite_node(put, #{bindings := #{node := Node0}}) ->
+invite_node(put, #{bindings := #{node := Node0}, body := Body}) ->
+    Node = ekka_node:parse_name(binary_to_list(Node0)),
+    case maps:get(<<"timeout">>, Body, ?DEFAULT_INVITE_TIMEOUT) of
+        T when not is_integer(T) ->
```

Comment:
Nit: could that be verified with schema?

Reply:
In theory, it really should. But for the moment, I haven't found a suitable example to check the format of a field request via schema. 😭